packages/lodestar-cli/src/cmds/beacon/config.ts

```diff
@@ -30,9 +31,14 @@ export async function writeBeaconConfig(filename: string, config: Partial<IBeaco
 
 /**
  * This needs to be a synchronous function because it will be run as part of the yargs 'build' step
+ * If the config file is not found, the default values will apply.
  */
 export function readBeaconConfig(filename: string): Partial<IBeaconNodeOptions> {
-  return readFileSync(filename) as Partial<IBeaconNodeOptions>;
+  if (fs.existsSync(filename)) {
+    return readFileSync(filename) as Partial<IBeaconNodeOptions>;
+  } else {
+    return {};
+  }
```

Comment:
Letting the config file not exist is necessary to allow running `run` without running `init` first, which may not necessary anyway.